tools/tcpdrop.py

```diff
@@ -193,6 +200,16 @@
 else:
     bpf_text = bpf_text.replace('FILTER_FAMILY', '')
 
+if args.pid_netns != 0:
+    args.netns_id = os.stat('/proc/{}/ns/net'.format(args.pid_netns)).st_ino
```

Comment:
Let us add a check here. If both pid_netns and netns_id are not zero, return a failure.